### PR TITLE
Improve edit media screen

### PR DESCRIPTION
**Pull request by yahilmadakiya**

### CODE REVIEW

File: app/assets/css/sass/_img-edit.scss [hunk 1]
```diff
@@ -511,7 +542,7 @@ td.imgedit-settings input {
 }
 
 .imgedit-crop {
-	background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -9px -31px;
+	// background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -9px -31px;
```
Comment (milindmore22) [inline app/assets/css/sass/_img-edit.scss]: please remove commented code .

File: app/assets/css/sass/_img-edit.scss [hunk 2]
```diff
@@ -524,7 +555,7 @@ td.imgedit-settings input {
 }
 
 .imgedit-rleft {
-	background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -46px -31px;
+	// background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -46px -31px;
```
Comment (milindmore22) [inline app/assets/css/sass/_img-edit.scss]: please remove commented code

File: app/assets/css/sass/_img-edit.scss [hunk 3]
```diff
@@ -537,7 +568,7 @@ td.imgedit-settings input {
 }
 
 .imgedit-rright {
-	background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -77px -31px;
+	// background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -77px -31px;
```
Comment (milindmore22) [inline app/assets/css/sass/_img-edit.scss]: please remove commented code

File: app/assets/css/sass/_img-edit.scss [hunk 4]
```diff
@@ -550,7 +581,7 @@ td.imgedit-settings input {
 }
 
 .imgedit-flipv {
-	background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -115px -31px;
+	// background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -115px -31px;
```
Comment (milindmore22) [inline app/assets/css/sass/_img-edit.scss]: please remove commented code

File: app/assets/css/sass/_img-edit.scss [hunk 5]
```diff
@@ -563,7 +594,7 @@ td.imgedit-settings input {
 }
 
 .imgedit-fliph {
-	background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -147px -31px;
+	// background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -147px -31px;
```
Comment (milindmore22) [inline app/assets/css/sass/_img-edit.scss]: please remove commented code

File: app/assets/css/sass/_img-edit.scss [hunk 6]
```diff
@@ -576,7 +607,7 @@ td.imgedit-settings input {
 }
 
 .imgedit-undo {
-	background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -184px -31px;
+	// background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -184px -31px;
```
Comment (milindmore22) [inline app/assets/css/sass/_img-edit.scss]: please remove commented code

File: app/assets/css/sass/_img-edit.scss [hunk 7]
```diff
@@ -589,7 +620,7 @@ td.imgedit-settings input {
 }
 
 .imgedit-redo {
-	background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -215px -31px;
+	// background: transparent url('../../../../../../wp-admin/images/imgedit-icons.png') no-repeat scroll -215px -31px;
```
Comment (milindmore22) [inline app/assets/css/sass/_img-edit.scss]: please remove commented code

File: app/assets/css/sass/_img-edit.scss [hunk 8]
```diff
@@ -676,27 +707,42 @@ span.imgedit-scale-warn {
...
+	border-width: 1px;
+	border-style: solid;
 }
 
-.bp-media-image-editor-cotnainer div.updated p { margin: .5em 0;padding: 2px; }
\ No newline at end of file
+.bp-media-image-editor-cotnainer div.updated p {
+	margin: .5em 0;
+	padding: 2px;
+}
```
Comment (milindmore22) [inline app/assets/css/sass/_img-edit.scss]: Add new line at end of file

**Comment by yahilmadakiya** [comment]
@milin all changes are done, can you please check once.

**Comment by milindmore22** [comment]
@PatelUtkarsh please do a final review

**Pull request merged**